torch/_inductor/fx_passes/reinplace.py

```diff
@@ -708,6 +709,7 @@ def tensor_with_same_storage_already_reinplaced(arg):
 
 
 def reinplace_inplaceable_ops(graph: torch.fx.Graph) -> None:
-    canonicalize_view_scatter_ops(graph)
-    reinplace_inplaceable_ops_core(graph)
-    decompose_generalized_scatter(graph)
+    with enable_python_dispatcher():
```

Comment:
Can we just decorate the entire function reinplace_inplaceable_ops?

Reply:
In fact, I'm kind of thinking we should decorate the entire compiler